executor/src/witgen/jit/variable.rs

```diff
@@ -42,11 +55,24 @@ impl Variable {
                 id: cell.id,
                 ptype: PolynomialType::Committed,
             }),
-            Variable::Param(_) => None,
+            Variable::Param(_) | Variable::MachineCallReturnValue(_) => None,
```

Comment:
This means that we'll never have a global range constraint on them. Fine for now, but in theory we could look into the RHS of the connection for that. Gets more complicated with a dynamic bus though.

Reply:
Yeah, maybe we could also just return constraints together with the "can always solve" call.